README.md

```diff
@@ -70,7 +70,7 @@ Alternatively, if your project has a `Package.swift` file, you can add AEPCore d
 
 ```
 dependencies: [
-    .package(url: "https://github.com/adobe/aepsdk-core-ios.git", .branch("main"))
+    .package(url: "https://github.com/adobe/aepsdk-core-ios.git", .upToNextMajor(from: "3.0.0"))
```

Comment:
https://github.com/adobe/aepsdk-core-ios/blob/a56d7878dbee82ab6edfd4ff67aae970b69527e5/Package.swift#L29 
Should we also update it with `upToNextMajor` statement?

Reply:
yeah that's a good idea, not needed for this release though.